olp-cpp-sdk-authentication/src/TokenProviderImplPrivate.h

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
Same, i think it is ok to put this in TokenProvider.cpp

Reply:
Moved implementation of TokenProviderImplPrivate to TokenProvider.cpp